_tabs/series.md

```
@@ -20,132 +20,132 @@ If you are new here, start with one cluster, follow its core path in order, and
+- [Practice: Kernel Build]({% link _posts/os/2025-03-09-OS(prac_1).md %})
 
 ## Computer architecture
 
 Learn how instructions, registers, memory, processors, pipelines, and caches fit together from a machine-level point of view.
 
 - **Intended reader level**: beginner to intermediate
-- **Recommended first post**: [Chapter 1-1: MIPS]({% link _posts/2025-03-11-CA(1-1).md %})
+- **Recommended first post**: [Chapter 1-1: MIPS]({% link _posts/ca/2025-03-11-CA(1-1).md %})
 - **Related clusters**: [Operating system](#operating-system), [Computer networking and internet protocol](#computer-networking-and-internet-protocol)
 
 ### Core path
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)